pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
Sometimes you have to. UHS30 use sblocking ISRs that can have delays up to 30 seconds... Since they run with interrupts open and low priority, other handlers are happily serviced, and is unaffected. The main code, yep, it sits there, but that doesn't matter since it doesn't matter. I never ever do anything time sensitive in a main loop, I always use a higher priority timer.